MvvmHelpers/WeakEventManager.cs

```diff
@@ -5,9 +5,10 @@
 
 using static System.String;
 
+[assembly: InternalsVisibleTo("MvvmHelpers.UnitTests")]
 namespace MvvmHelpers
 {
-	public class WeakEventManager
+	internal class WeakEventManager
```

Comment:
Is there a reason we shouldn't make this public? Seems like it could be useful for library creators.

Reply:
I read the comment now... hmmmmmm interesting....

Reply:
Yes, actually that makes a lot of sense because I want to use the simplest solution in my apps, and I hope that's the case of a lot of other developers too and it's an extra piece of code that needs to be maintained and needs improvement over time.

Reply:
I think I am fine keeping it public :)

Reply:
Okay I change it to public